### ISSUE TITLE
The file menu is getting a bit crowded

### ISSUE THREAD

**Issue by RetroEdit** [mgba-emu/mgba#2126]
I count 28, more than any other menu in mGBA. It might be wise to condense/combine some items. File menu fatigue is a real thing. :smiley: 

![mGBA_file_menu_clutter](https://user-images.githubusercontent.com/30182911/114869343-622bab00-9de6-11eb-8851-f70bbae55d6f.png)

**Comment by endrift** [comment]
Yeah I'd been noticing this too. I'm probably gonna move the convert e-Reader card option to tools, and the GameShark save things will likely be rolled into the save game converter.

When there are more debug tools I'm probably gonna split the tools menu out too.

**Comment by endrift** [comment]
I'd also like a better (shorter) phrasing than "Convert e-Reader card image to raw..." but I couldn't think of anything

**Comment by endrift** [comment]
I moved "make portable" to the tools menu, right under settings, and "load camera image" to the emulation menu with the other game-specific hardware options. Having a hard time figuring out other stuff that should be moved.

**Comment by RetroEdit** [comment]
Maybe a separate `Help` menu on the top with `Report bug...` and `About...`?

Maybe a separate `Game save` submenu for all the game save-related items (`Load alternate save game...`, `Load temporary save game...`, `Convert save game...`, `Import GameShark Save...`, `Export GameShark Save...`; the last two might not fit with the others)?

There could be a separate `e-reader` submenu, though two items is a little sparse. It would solve the text-width issue.

I do wonder if submenus with distinct actions inside (as opposed to value-based 10%, 20%, etc. submenus) appeal to your UI design aesthetics; I would imagine you would have already used submenus if they did, but the menu items have kind of crept up over time.

There are some other ideas I have, but I'll have to think about it some more.

**Comment by endrift** [comment]
Report bug is intentionally in the file menu so people see it, and a help menu with just one (or two) items seems a bit lackluster.

Game save submenu may work, though I'm hoping to fold the gameshark save stuff into convert save dialog in 0.10.

What do you mean by "distinct actions instead of values" though? A lot of the value-heavy menus already have related-but-distinct actions in them.

**Comment by RetroEdit** [comment]
> Report bug is intentionally in the file menu so people see it

I would argue a new `Help` menu, even with only two items, is much more visible. Having a `Help` menu with these sort of things is a common UI paradigm that people probably subconsciously pick up on. But it's easy to miss buried in a menu with 20+ items.

But I suppose the counterargument is if the `File` menu was significantly trimmed, it would be more visible in the `File` menu.

> What do you mean by "distinct actions instead of values" though?

Eh, it would be hard to explain without making an exhaustive list of menus and how they're laid out in mGBA. Maybe I had a biased sample, but most of the submenus I looked at were mainly values, with one or two separate actions.

I do think in some cases, like the GameShark save stuff, a separate dialog is probably the way to go.

**Comment by endrift** [comment]
I think the primary reason there aren't many other submenus is that there just...aren't enough similar items to warrant one without totally emptying out the menu (as would happen in the tools menu, though I want to turn that into the debug menu when that happens anyway).
